content/200-orm/200-prisma-client/100-queries/037-relation-queries.mdx

```diff
@@ -26,7 +26,7 @@ Nested reads allow you to read related data from multiple tables in your databas
 
 ### Relation load strategies (Preview)
 
-Since version [5.9.0](https://github.com/prisma/prisma/releases/tag/5.9.0), you can decide on a per-query-level _how_ you want Prisma Client to execute a relation query (i.e. what _load strategy_ should be applied) via the `relationLoadStrategy` option.
+Since version [5.10.0](https://github.com/prisma/prisma/releases/tag/5.10.0), you can decide on a per-query-level _how_ you want Prisma Client to execute a relation query (i.e. what _load strategy_ should be applied) via the `relationLoadStrategy` option.
```

Comment:
Had to go back and check our internal communication about this. Definitely a mistake in our docs. Hopefully this suggestion makes things clearer :)

```suggestion
Since version [5.8.0](https://github.com/prisma/prisma/releases/tag/5.8.0), you can decide on a per-query-level _how_ you want Prisma Client to execute a relation query (i.e. what _load strategy_ should be applied) via the `relationLoadStrategy` option for PostgreSQL databases.

Since version [5.10.0](https://github.com/prisma/prisma/releases/tag/5.10.0), this feature is also available for MySQL.
```

Reply:
Yes makes it clearer